### DISCUSSION TITLE
Is it possible to ignore some dependencies from checking?

### DISCUSSION THREAD

**Discussion by jskiba** [dominikh/go-tools#1241] [Q&A] [answered]
I use [https://github.com/godror/godror](url) in my project and the problem is that it does not pass staticcheck but only when GOOS=windows is set. So I would like to make staticcheck ignore it, is there some way to do so?

**Comment by dominikh** [comment, marked as answer]
Staticcheck doesn't check your dependencies, however it does require them to compile correctly. There is no way around that. That package uses cgo, so you can't trivially cross-compile it just by setting a different GOOS. You will have the same issue trying to `go build` your project.

**Reply by jskiba** [reply]
but if I set CC to mingw then I should be able to build binaries for windows

**Reply by dominikh** [reply]
If you are able to set up proper cross compilation that works for `go build`, then using the same environment variables should work for Staticcheck.

**Reply by linnjason678** [reply]
> but if I set CC to mingw then I should be able to build binaries for windows

Why ingnore,?

**Comment by linnjason678** [comment]
> I use [https://github.com/godror/godror](url) in my project and the problem is that it does not pass staticcheck but only when GOOS=windows is set. So I would like to make staticcheck ignore it, is there some way to do so?

Why ingnore,  that seems to cause the issues.
